### ISSUE TITLE
Field handler is trigger even if the field does not exist in the resource at the time.

### ISSUE THREAD

**Issue by euan-tilley** [zalando-incubator/kopf#375]
## Long story short

Field handler is trigger even if the field doesn't exist in resource

## Description

When we create a customer resource the create handler is correctly triggered but the a field handler is also triggered for fields that don't exist in the resource when it was created.
Not sure if this is a bug in 0.27 or a feature but it is a change in functionality from previous versions. 

<details><summary>The code snippet to reproduce the issue</summary>

```python
@kopf.on.create('company.com', 'v1', 'testresources')
def create_handler(spec, **_):
    pass

@kopf.on.field('company.com', 'v1', 'testresources', field='spec.updateField')
def updateField_changed(old, new, **_):
    pass

#Custom Resource that was created
apiVersion: company.com/v1
kind: TestResource
metadata:
  name: test
spec:
  createField: true
```
</details>

<details><summary>The exact command to reproduce the issue</summary>

```bash
kopf run --standalone test-handler.py
```
</details>

<details><summary>The full output of the command that failed</summary>

```
[2020-06-13 20:13:16,055] kopf.activities.auth [INFO    ] Activity 'login_via_client' succeeded.
[2020-06-13 20:13:16,055] kopf.reactor.activit [INFO    ] Initial authentication has finished.
[2020-06-13 20:13:29,639] kopf.objects         [INFO    ] [default/test] Handler 'create_handler' succeeded.
[2020-06-13 20:13:29,760] kopf.objects         [INFO    ] [default/test] Handler 'updateField_changed/spec.updateField' succeeded.
[2020-06-13 20:13:29,760] kopf.objects         [INFO    ] [default/test] All handlers succeeded for creation.
```
</details>

## Environment

* Kopf version: 0.27
* Kubernetes version: minikube v0.35.0 on darwin (amd64) & v.1.11.9
* Python version: 3.8.0
* OS/platform: macos & coreos

<details><summary>Python packages installed</summary>

```
aiohttp==3.6.2
aiojobs==0.2.2
async-timeout==3.0.1
attrs==19.3.0
cachetools==4.1.0
certifi==2020.4.5.2
cffi==1.14.0
chardet==3.0.4
click==7.1.2
cryptography==2.9.2
google-auth==1.17.2
idna==2.9
iso8601==0.1.12
Jinja2==2.11.2
kopf==0.26
kubernetes==11.0.0
MarkupSafe==1.1.1
multidict==4.7.6
oauthlib==3.1.0
pip==19.3.1
pyasn1==0.4.8
pyasn1-modules==0.2.8
pycparser==2.20
pycrypto==2.6.1
pykube-ng==20.5.0
pymongo==3.10.1
pyOpenSSL==19.1.0
python-dateutil==2.8.1
PyYAML==5.3.1
requests==2.23.0
requests-oauthlib==1.3.0
rsa==4.6
setuptools==41.6.0
six==1.15.0
typing-extensions==3.7.4.2
urllib3==1.25.9
websocket-client==0.57.0
wheel==0.33.6
yarl==1.4.2
```
</details>

**Comment by smileisak** [comment]
`+1` Experiencing the same issue. It is only present in `v0.27` not in the previous versions.

**Comment by nolar** [comment]
Solved in https://github.com/nolar/kopf/pull/523, available since `kopf>=0.28rc2`
